tests/lib/forms-schema.test.js

```diff
@@ -4201,3 +4267,22 @@ test('should allow placeholderValue property for these elements', () => {
   )
   expect(result.error).toBe(null)
 })
+
+test.only('should allow forms without tags', () => {
+  const result = Joi.validate(
+    {
+      id: 1,
+      name: 'Tags Form',
+      formsAppEnvironmentId: 1,
+      formsAppIds: [1],
+      organisationId: '59cc888b8969af000fb50ddb',
+      postSubmissionAction: 'FORMS_LIBRARY',
+      isMultiPage: false,
+      submissionEvents: [],
+      elements: [],
+    },
+    formSchema,
+  )
+  expect(result.value.tags === [])
```

Comment:
I dont think this is doing what you want it to:

```suggestion
  expect(result.value.tags).toEqual([])
```

https://jestjs.io/docs/en/expect#toequalvalue